Alignment/CommonAlignmentProducer/python/AlcaSiPixelAliHarvesterHG_cff.py

```diff
@@ -0,0 +1,79 @@
+import FWCore.ParameterSet.Config as cms
+import copy
+
+SiPixelAliMilleFileExtractorHG = cms.EDAnalyzer("MillePedeFileExtractor",
+    fileBlobInputTag = cms.InputTag("SiPixelAliMillePedeFileConverterHG",''),
+    fileDir = cms.string('HGalignment/'),
+    # File names the Extractor will use to write the fileblobs in the root
+    # file as real binary files to disk, so that the pede step can read them.
+    # This includes the formatting directive "%04d" which will be expanded to
+    # 0000, 0001, 0002,...
+    outputBinaryFile = cms.string('pedeBinaryHG%04d.dat'))
+
+from Alignment.MillePedeAlignmentAlgorithm.MillePedeAlignmentAlgorithm_cfi import *
+from Alignment.CommonAlignmentProducer.AlignmentProducerAsAnalyzer_cff import AlignmentProducer
+SiPixelAliPedeAlignmentProducerHG = copy.deepcopy(AlignmentProducer)
+
+from Alignment.MillePedeAlignmentAlgorithm.MillePedeDQMModule_cff import *
+
+
+SiPixelAliPedeAlignmentProducerHG.ParameterBuilder.Selector = cms.PSet(
+    alignParams = cms.vstring(
+        "TrackerP1PXBLadder,111111",
+        "TrackerP1PXECPanel,111111",
+        )
+    )
+
+SiPixelAliPedeAlignmentProducerHG.doMisalignmentScenario = False #True
+
+SiPixelAliPedeAlignmentProducerHG.checkDbAlignmentValidity = False
+SiPixelAliPedeAlignmentProducerHG.applyDbAlignment = True
+SiPixelAliPedeAlignmentProducerHG.tjTkAssociationMapTag = 'TrackRefitter2'
+
+SiPixelAliPedeAlignmentProducerHG.algoConfig = MillePedeAlignmentAlgorithm.clone()
+SiPixelAliPedeAlignmentProducerHG.algoConfig.mode = 'pede'
```

Comment:
any reason why the parameters cannot be changed directly within the `clone` function?

Reply:
in general wherever possible I would say.